### ISSUE TITLE
User should be able to edit information collected through mind maps//Deblurbify

### ISSUE THREAD

**Issue by nehav39** [Intelehealth/Android-Mobile-Client#266]
Currently on the Visit Summary screen and on the Patient Detail screen, if the user needs to edit any information on a card, there are only two options, editing the text itself or erasing all text and recollecting this information from the mind maps. This is because we save the mind map output as a text blurb and not as a sequence of nodes.

The best way to do this is to store the mind map generated history note as a sequence of nodes so that these can be "reloaded" into the mind map and the user will be able to go in and edit only a certain node and not have to redo the entire data collection or make a manual edit.

This is related to issues #211 #213
 
In other words - "deblurbify!!!"
